assets/section-multicolumn.css

```diff
@@ -133,10 +132,6 @@
   margin: 0;
 }
 
-.multicolumn-card__info {
-  padding: 2.5rem 2.5rem;
-}
```

Comment:
We need this padding, at least when the columns have background https://screenshot.click/17-42-9msqq-ahqmj.png

Reply:
> We need this padding, at least when the columns have background https://screenshot.click/17-42-9msqq-ahqmj.png

**+1** on needing padding for [this](https://user-images.githubusercontent.com/28404165/158902574-9a1bfb89-29eb-4be2-b3b4-ae0058d28b19.png).

We could also change the value to just `2.5rem`:

```css
.multicolumn-card__info {
  padding: 2.5rem;
}
```

Reply:
Good catch, added it back and changed the value.